DiscordChatExporter.Cli/DiscordChatExporter.Cli.csproj

```diff
@@ -7,6 +7,7 @@
         <Company>Tyrrrz</Company>
         <Copyright>Copyright (c) 2017-2018 Alexey Golub</Copyright>
         <ApplicationIcon>..\favicon.ico</ApplicationIcon>
+        <NoWin32Manifest>true</NoWin32Manifest>
```

Comment:
What's the purpose of this?

Reply:
Automated change by visual studio, I'll remove it.